airbyte-webapp/src/pages/DestinationPage/pages/DestinationItemPage/components/DestinationSettings.tsx

```diff
@@ -4,21 +4,23 @@ import styled from "styled-components";
 
 import DeleteBlock from "components/DeleteBlock";
 
-import { Connection, ConnectionConfiguration } from "core/domain/connection";
+import { ConnectionConfiguration } from "core/domain/connection";
 import { Connector, Destination } from "core/domain/connector";
 import { useDeleteDestination, useUpdateDestination } from "hooks/services/useDestinationHook";
 import { useDestinationDefinition } from "services/connector/DestinationDefinitionService";
 import { useGetDestinationDefinitionSpecification } from "services/connector/DestinationDefinitionSpecificationService";
 import { ConnectorCard } from "views/Connector/ConnectorCard";
 
+import { WebBackendConnectionRead } from "../../../../../core/request/AirbyteClient";
```

Comment:
Unnecessary relative path import. I've seen this in a few places, actually.

Reply:
I've since adjusted my vs code settings. I can try to find all the spots and fix if you feel strongly.

Reply:
This particular one caught my attention because it's sorted in the wrong place.

Reply:
Fixed!